mono/mini/optflags-def.h

```diff
@@ -21,9 +21,8 @@ OPTFLAG(SSAPRE   ,19, "ssapre",     "SSA based Partial Redundancy Elimination (o
 OPTFLAG(EXCEPTION,20, "exception",  "Optimize exception catch blocks")
 OPTFLAG(SSA      ,21, "ssa",        "Use plain SSA form")
 OPTFLAG(SSE2     ,23, "sse2",       "SSE2 instructions on x86")
```

Comment:
Index 22 is missing which causes issues on windows with opmizations defined after this line. Chaging to 
```
OPTFLAG(SSE2     ,22, "sse2",       "SSE2 instructions on x86")
OPTFLAG(GSHAREDVT,23, "gsharedvt",	"Generic sharing for valuetypes")
OPTFLAG (GSHARED, 24, "gshared", "Generic Sharing")
OPTFLAG(SIMD	 ,25, "simd",	    "Simd intrinsics")
...
```
works.